vitest.workspace.ts

```diff
@@ -18,7 +18,7 @@ export const defaultTypeSpecVitestConfig = defineConfig({
     outputFile: {
       junit: "./test-results.xml",
     },
-    exclude: ["node_modules", "dist/test"],
+    exclude: ["node_modules", "dist/**/*.test.js"],
```

Comment:
```suggestion
    exclude: ["node_modules", "dist/**/*.test.*"],
```

might just want to do that so it exclude other things like `jsx`